src/AnimalFamily/AnimalFamily.ts

```diff
@@ -58,7 +58,7 @@ export default class AnimalFamily {
         );
         const verifycationResult: VerificationResult =
           verifyFamilyInfo(specificRequirements);
-        if (verifycationResult.isValid) {
+        if (verifycationResult.message === "") {
```

Comment:
you can extract in a constant `const isVerificationValid = verifycationResult.message === ""`